config/routes.rb

```diff
@@ -1,4 +1,12 @@
 Rails.application.routes.draw do
+  # get 'pages/hello'
+  # root 'pages#hello'
+  root 'users#index'
+  resources :users, only: [:index, :show] do
+    resources :posts, only: [:index, :show, :new, :create] do
```

Comment:
- [ ] Hey there! 🌟 Taking into account the previous feedback, let's streamline our focus for now. We'll only include routes that are essential at this stage—specifically, routes for posts#index and posts#show. 🚀 Your attention to this detail is much appreciated! 🙌

```ruby
# config/routes.rb

  resources :users, only: [:index, :show] do
    resources :posts, only: [:index, :show]
  end
```